keps/sig-node/3317-user-namespaces/README.md

```diff
@@ -0,0 +1,1015 @@
+# KEP-3317: Support User Namespaces
```

Comment:
Can you clarify why we're effectively creating a new enhancement issue for the same thing?

Why can't we continue using the 127 issue and KEp number?

Reply:
as you mentioned in the tracking issue you strongly prefer to keep the KEP number and offered to update the old issue (which any of the KEP authors can edit), I'm keeping the same KEP number now.